packages/wrangler/src/bundle.ts

```diff
@@ -457,16 +459,18 @@ export async function bundleWorker(
 		entryPointOutputs[0][0]
 	);
 
+	const modules = [...additionalModules, ...moduleCollector.modules];
```

Comment:
Is it possible that `additionalModules` and `moduleCollector.modules` contain duplicates?
Do we care? Probably not here, since all we are doing is writing them out to the directory containing the entry-point, so we just waste some time writing files more than once.

But this `modules` array is returned and could that result in multiple copies of the same file being added to an HTTP Form?

Reply:
HTTP form defaults automatically de-dupes by preferring the last entry. Workerd crashed. I added this commit to more gracefully handle the unlikely case of a collision: https://github.com/cloudflare/workers-sdk/pull/3175/commits/742dee496dea9e2451212c59d243aee9cf2abe6a